RELEASE_NOTES.md

```diff
@@ -19,7 +19,56 @@
 
 Version v0.2.0 -- Release date 2020-06-20
 ==========================================
-Lots of stuff to fill in...
+* Extensive updates to Data Repository.
+* Discretization:
+  * Finite element interface
+    * Kernel launching interface looping abstraction.
+    * Element formulations for 1st order:
+      * 8 node hexahedron
+      * 6 node wedge
+      * 5 node pyramid
+      * 4 node tetrahedron
+  * Finite volume stencils for classical (cell-centered) FVM formulation (TPFA, infrastructure support for MPFA, but currently 
+    not implemented)
+  * Inner products for Hybrid FVM formulation (current support for TPFA and quasi-TPFA) 
+* Physics Solvers
+  * Solid mechanics explicit on GPU, implicit assembly on GPU
+  * Single phase flow (assembly on GPU)
+    * Classical FVM and Hybrid FVM formulations
+    * Porous matrix and DFM fracture flow
+  * Compositional multiphase flow (assembly on GPU)
+  * Multi-segmented wells for single phase and compositional multiphase flow (assembly on GPU)
+    * Fully implicit isothermal overall composition formulation
+    * A library of fluid constitutive models:
+      * Equation-of-state based hydrocarbon compositional
+      * Three-phase extended black-oil
+      * Two-phase CO2-brine  
```

Comment:
The order of bullet points got messed up a bit
```suggestion
  * Compositional multiphase flow (assembly on GPU)
    * Fully implicit isothermal overall composition formulation
    * A library of fluid constitutive models:
      * Equation-of-state based hydrocarbon compositional
      * Three-phase extended black-oil
      * Two-phase CO2-brine
  * Multi-segmented wells for single phase and compositional multiphase flow (assembly on GPU)
```

Reply:
@klevzoff can you re-suggest this? It is out of date and I can't commit it.